perf_test/sparse/KokkosSparse_sptrsv_superlu.cpp

```diff
@@ -220,21 +234,39 @@ void factor_superlu (bool symm_mode, bool metis,
   printf( "  + calling SuperLU dgstrf with panel_size=%d, relax_size=%d..\n",panel_size,relax_size );
   printf( "   * Dimension %dx%d; # nonzeros %d\n", A.nrow, A.ncol, Astore->nnz);
 
+  #ifdef HAVE_KOKKOSKERNELS_SUPERLU5_API
```

Comment:
@iyamazaki Just to make sure I understand: Do we need to  set ```-DHAVE_KOKKOSKERNELS_SUPERLU5_API:BOOL=ON``` in the cmake command when building Kokkos Kernels?

Reply:
Hi, @vqd8a. I am not sure that will work.. For my testing, I just added `-DHAVE_KOKKOSKERNELS_SUPERLU5_API` to `CMAKE_CXX_FLAGS`.. It will be nice if there is a cleaner way like that (or can configure figure out SuperLU version?).

Reply:
Thanks @iyamazaki . I am fine with setting -DHAVE_KOKKOSKERNELS_SUPERLU5_API in CMAKE_CXX_FLAGS.